app/src/layouts/AppLayout/index.tsx

```diff
@@ -40,6 +41,7 @@ const App: React.FC = () => {
     growthbook.loadFeatures({ autoRefresh: true });
   }, []);
 
+  useLastFeatureTracker();
```

Comment:
This places it above the [`RootComponent`](https://github.com/requestly/requestly/blob/master/app/src/components/redirects/RootComponent.tsx#L17) that has its own sets of redirects

That component is where the [`redirectToRoot`](https://github.com/requestly/requestly/blob/master/app/src/utils/RedirectionUtils.js) expects to land. Just make sure that does not break (I have done basic testing for this, so far it works well)